apps/redi-talent-pool/src/pages/front/signup/SignUp.tsx

```diff
@@ -219,8 +219,11 @@ export default function SignUp() {
         <Columns.Column size={5} offset={1}>
           <Heading border="bottomLeft">Sign-up</Heading>
           <Content size="small" renderAs="p">
-            Got a ReDI Connect user account? You can log in with the same
-            username and password <Link to="/front/login">here</Link>.
+            {/* Commented and replaced with different text until the cross-platform log-in feature is implemented. */}
+            {/* Got a ReDI Connect user account? You can log in with the same
+            username and password <Link to="/front/login">here</Link>. */}
+            Got a ReDI Connect user account? To log in with the same username 
+            and password get in contact with @Kate in ReDI Slack or write an e-mail to kateryna@redi-school.org.
```

Comment:
Here and elsewhere you can improve this slightly by making it into a link:
`<a href="mailto:kateryna@redi-school.org>kateryna@redi-school.org</a>`

Reply:
I didn't want to attract that much attention, but I probably should:)